src/WebApi/Controllers/UsersController.cs

```diff
@@ -210,6 +210,38 @@ public Task<ActionResult<Result<CharacterCharacteristicsViewModel>>> GetCharacte
         }));
     }
 
+    /// <summary>
+    /// Get character competitive rating.
+    /// </summary>
+    /// <param name="id">Character id.</param>
+    /// <returns>The character rating.</returns>
+    /// <response code="200">Ok.</response>
+    [HttpGet("self/characters/{id}/competitiverating")]
```

Comment:
it is common to use the Kebab Case
`/competitiverating` -> `/competitive-rating`